kubernetes/resource_kubernetes_resource_priority_class_test.go

```diff
@@ -0,0 +1,257 @@
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.labels.TestLabelOne", "one"),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.labels.TestLabelThree", "three"),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.labels.TestLabelFour", "four"),
+					testAccCheckMetaLabels(&conf.ObjectMeta, map[string]string{"TestLabelOne": "one", "TestLabelThree": "three", "TestLabelFour": "four"}),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.name", name),
+					resource.TestCheckResourceAttrSet("kubernetes_priority_class.test", "metadata.0.generation"),
+					resource.TestCheckResourceAttrSet("kubernetes_priority_class.test", "metadata.0.resource_version"),
+					resource.TestCheckResourceAttrSet("kubernetes_priority_class.test", "metadata.0.self_link"),
+					resource.TestCheckResourceAttrSet("kubernetes_priority_class.test", "metadata.0.uid"),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "value", "100"),
+				),
+			},
+			{
+				Config: testAccKubernetesPriorityClassConfig_metaModified(name),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckKubernetesPriorityClassExists("kubernetes_priority_class.test", &conf),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.annotations.%", "2"),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.annotations.TestAnnotationOne", "one"),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.annotations.TestAnnotationTwo", "two"),
+					testAccCheckMetaAnnotations(&conf.ObjectMeta, map[string]string{"TestAnnotationOne": "one", "TestAnnotationTwo": "two"}),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.labels.%", "3"),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.labels.TestLabelOne", "one"),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.labels.TestLabelTwo", "two"),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.labels.TestLabelThree", "three"),
+					testAccCheckMetaLabels(&conf.ObjectMeta, map[string]string{"TestLabelOne": "one", "TestLabelTwo": "two", "TestLabelThree": "three"}),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "metadata.0.name", name),
+					resource.TestCheckResourceAttrSet("kubernetes_priority_class.test", "metadata.0.generation"),
+					resource.TestCheckResourceAttrSet("kubernetes_priority_class.test", "metadata.0.resource_version"),
+					resource.TestCheckResourceAttrSet("kubernetes_priority_class.test", "metadata.0.self_link"),
+					resource.TestCheckResourceAttrSet("kubernetes_priority_class.test", "metadata.0.uid"),
+					resource.TestCheckResourceAttr("kubernetes_priority_class.test", "value", "100"),
```

Comment:
Updates to this field should be tested.

Reply:
Value can't be changed, so there is no need for this, right?

Reply:
Same comment about the file name.